### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by opustecnica** [nalgeon/sqlean#121]
Anton,
would it be possible to include the 'expert' extension as a loadable one either individually or as part of the sqlean package?

Thank you

**Comment by nalgeon** [comment]
Do you mean the [SQLite Expert Extension](https://www.sqlite.org/src/dir?ci=trunk&name=ext/expert)? It has no SQL API, only C or command line. How do you want to use it, can you give a concrete example?

**Comment by opustecnica** [comment]
Would love to use it as a loadable extension when NOT using sqlite3 (e.g. powershell script leveraging system.data.sqlite).
System.data.sqlite allows for on-the-fly creation of functions (in .NET) that are not covered by other traditional/available extensions. 
By way of example, earlier today, in an attempt to utilize the .expert, I started to replicate a query in sqlite3 only to realize I was missing a function I had created in .NET to transform a string containing a date (''Jan 19, 2022 12:00:00 UTC'' into '2022-01-19 12:00:00') to a string that was datetime() compatible ... obviously more a matter of convenience rather than absolute need.
